app/utils/answersHelpers/consignment/HouseConsignmentAnswersHelper.scala

```diff
@@ -158,6 +154,28 @@ class HouseConsignmentAnswersHelper(
         )
     }
 
+  def additionalReferenceSection: Section =
+    userAnswers.get(AdditionalReferenceListSection(houseConsignmentIndex)).mapWithIndex {
+      case (_, index) =>
+        val helper = new HouseConsignmentAdditionalReferencesAnswersHelper(userAnswers, houseConsignmentIndex, index)
+        val rows = Seq(
+          helper.referenceType,
+          helper.referenceNumber
+        ).flatten
+        AccordionSection(
+          sectionTitle = Some(messages("unloadingFindings.houseConsignment.additionalReference", index.display)),
+          rows = rows,
+          id = Some(s"additionalReference$index")
+        )
+    } match {
+      case children =>
+        AccordionSection(
+          sectionTitle = Some(messages("unloadingFindings.houseConsignment.additionalReference.heading")),
+          children = children,
+          id = Some("additionalReference")
```

Comment:
Very very minor but additionalReferences would make more sense here. Same with the name of the def (additionalReferencesSection instead of additionalReferenceSection)